lib/vast/Conversion/Common/IRsToLLVM.cpp

```diff
@@ -426,10 +426,33 @@ namespace vast::conv::irstollvm
 
         }
 
+        logical_result handle_void_const(
+                hl::ConstantOp op, conversion_rewriter &rewriter) const
+        {
+            auto result = op.getResult();
+            if (result.hasOneUse()) {
+                auto user = result.getUses().begin()->getOwner();
+                if (is_one_of_mlir<hl::ReturnOp, ll::ReturnOp, core::ImplicitReturnOp >(
```

Comment:
we should have ReturnTrait for this ... or at least a typelist at a single location